### PR TITLE
docs(components): [a11y] add tag

### PR DESCRIPTION
**Pull request by chenxch**
- Add A11yTag

light:
image

dark:
image

Please make sure these boxes are checked before submitting your PR, thank you!

- [ ] Make sure you follow contributing guide [English](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.en-US.md) | ([中文](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.zh-CN.md) | [Español](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/element-plus/element-plus/blob/master/.github/CONTRIBUTING.fr-FR.md)).
- [ ] Make sure you are merging your commits to `dev` branch.
- [ ] Add some descriptions and refer to relative issues for your PR.

### CODE REVIEW

**Review by holazz — APPROVED** [review]

**Review by btea — APPROVED** [review]

**Review by ryuhangyeong — APPROVED** [review]

File: docs/.vitepress/vitepress/components/dev/A11yTag.vue [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<template>
+  <el-tag size="small" effect="plain" hit round class="ml-8px">a11y</el-tag>
```
Comment (jw-foss) [inline docs/.vitepress/vitepress/components/dev/A11yTag.vue:2]: I think we can use `ml-3` here for representing `8px`

File: docs/.vitepress/vitepress/components/dev/A11yTag.vue [hunk 2]
```diff
@@ -0,0 +1,13 @@
+<template>
```
Comment (jw-foss) [inline docs/.vitepress/vitepress/components/dev/A11yTag.vue:1]: For the file name, please follow the convention with **kebab-case** instead of PascalCase.

**Review by jw-foss — COMMENTED** [review]
@chenxch left a few comments for you to consider, let me know if you have any questions :D

**Comment by chenxch** [comment]
@jw-foss Need your review again.

**Review by jw-foss — APPROVED** [review]

**Comment by jw-foss** [comment]
Just a TODO here, we may move these color tokens into our CSS variable files later.

**Pull request merged**